cli/cmd/sudo/project/search.go

```diff
@@ -9,9 +9,10 @@ import (
 func SearchCmd(ch *cmdutil.Helper) *cobra.Command {
 	var pageSize uint32
 	var pageToken string
+	var tags []string
 
 	searchCmd := &cobra.Command{
-		Use:   "search",
+		Use:   "search <pattern>",
 		Args:  cobra.ExactArgs(1),
```

Comment:
Let's make the pattern optional, like `search [<pattern>]` and have it default to `%` if not set (which means match all)